common/pipeline_storage/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+sqs:
+  image: s12v/elasticmq
+  ports:
+    - "9324:9324"
```

Comment:
I haven't read down yet, but I'm immediately curious about what a messaging service is doing in a storage module.

Reply:
Yes I also agree with this I think there is code in here which seems more specific than we would want for `pipeline_storage`. How about pulling out only some of the code, and leaving `ingestor_common` to have the service related stuff?

Reply:
I think that’d be much better – have `ingestor_common` depend upon this module, so the ingestors still get all its goodies, but the enricher-or-whatever-it’s-called-now doesn’t need the ingestor services.

Reply:
Ok will give it a go, hopefully quite easily untangled